flipperdevices/flipperzero-firmware
# Add Initial CCID support

# What's new

Adds a new CCID (Smartcard) subsystem, along with a test app

# Verification 

On Linux:
 - Add a new VID/PID pair in /usr/lib/pcsc/drivers/ifd-ccid.bundle/Contents/Info.plist. The one from the ccid_test app is VID=0x1234 and PID=0x5678
 - Build ccid_test FAP and install.
 - Run "sudo pcscd --foreground --debug"
 - Open ccid_test app
 - On pcscd logs you should see the following line:

```
Card ATR: 3B 00
```
 - Then, run the following command:

```
opensc-tool --send-apdu 01:02:03:04
```

You should see the following result:

```
Sending: 01 02 03 04 
Received (SW1=0x6E, SW2=0x00)

```


# Checklist (For Reviewer)
